authoring-service/src/main/resources/application.yml

```diff
@@ -8,11 +8,25 @@ spring:
       password: password
       database: survey
       port: 27017
+
+kafka:
+  bootstrapAddress: kafka:9092
```

Comment:
should we pass as env var?

Reply:
yes but seems out of scope for this branch, I'll create task in MS for that on project level